src/components/input/input.md

```diff
@@ -1,3 +1,11 @@
 ```jsx
-<Input id="greeting" placeholder="hello world" />
+<Input id="greeting" label="Greeting" />
+```
+
+```jsx
+<Input id="greeting" label="Greeting" invalid />
```

Comment:
what's `invalid` without a value mean here exactly?

Reply:
It's standard JSX. When you specify a value like this, it implies setting it as `true`. 

```
<Input id="greeting" label="Greeting" invalid={true} />
```